versioned_docs/version-latest/protocol/substrate-cli.mdx

```diff
@@ -551,22 +526,24 @@ We will be downloading two files for your respective operating system.
 
 </TabItem>
 
-<TabItem value="systemd_service" label="⚙ Service (Linux)">
+<TabItem value="systemd_service" label="⚙ Linux Server">
 
-:::warning Security risks
-Running a node as a root user carries significant risks for the system. 
-It is recommended to create a separate user for this purpose.
-:::
 
-<Tabs groudId="service_user">
-<TabItem value="root" label="🫚 Root" default>
-Create a Directory for Executable Files:
 
-```shell-session
-mkdir -p /root/.local/bin
+Adhering to standard Linux conventions, we place executable files in `/usr/local/bin`, create dedicated directories for their data, and run services from a special user.
+
+All further actions are performed under a regular user.
+
+### Creating a user with restricted rights to run the service
+```console
+sudo useradd --no-create-home --shell /bin/false subspace
```

Comment:
to restrict access without giving any feedback

Reply:
Well, it's ok to do it that way too, Either way I don't think we should try so hard to protect the participants' computers. They are just end users. It's enough just to do it according to the guidelines, common sense, etc. Not the way it should be done for production.

Reply:
Yes, it's not a bad collection. But would anyone go to that kind of effort to just break the testing of a network? There can be infinitely many security measures, but I think there is some limit of simplicity at which it would be simple enough for the mass user and yet secure enough.

Also, I think there needs to be more clarification - why you are recommending any particular settings. For example, even in your post Few optimizations for nodes and farmers you argue for your recommendations simply because that is your experience. Although there will be many inexperienced noderunners in our target audience who would prefer copy-paste, but I think it's better not to stick to the "do as I do" style, but detailed explanations.
So I don't mind taking something from there, but make it interesting.I'll analyze your site today, I think.